.flake8

```diff
@@ -0,0 +1,40 @@
+[flake8]
+exclude =
```

Comment:
I think we can also exclude also 
    .coverage,
    openIMIS/wsgi.py,
    dist,
    openIMIS/migrations